API/ControllerAttributes/MaxFileSizeAttribute.cs

```diff
@@ -0,0 +1,80 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Threading.Tasks;
+
+namespace API.ControllerAttributes
+{
+    /// <summary>
+    /// Attribute for settings the maximum allowed upload size for every single file in bytes in the request.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class MaxFileSizeAttribute : ActionFilterAttribute
+    {
+        // Max file size in bytes.
+        private readonly int maxFileSize;
+
+        /// <summary>
+        /// Constructor of the MaxFileSizeAttribute consumes maxFileSize in bytes
+        /// </summary>
+        /// <param name="maxFileSize">Max file size in bytes</param>
+        public MaxFileSizeAttribute(int maxFileSize)
+        {
+            this.maxFileSize = maxFileSize;
+        }
```

Comment:
This is not something that's 'wrong', but at first sight, the filesize in the controller seemed very strange to be in bytes (instead of kb or mb). If you have thought about this, then it's ok :smile: .

Reply:
I Assumed it to be the most flexible solution since that was requested.

Reply:
Sure! I was just curious. This is a good argument of course.